src/collectionHandlingFunctions.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+function describeCollectionItem(array $item): string {
+    
+    if($item===[]){
```

Comment:
Being nitpicky, there should be a space between the if and the (, and the ) and the {

`if ($item === []) {`

Reply:
done! (I also changed the other if statements)